spec/features/visitors/sign_up_spec.rb

```diff
@@ -52,20 +52,20 @@
     expect(user.reset_requested_at).to be_nil
   end
 
-  context 'visitor can sign up and confirm a valid mobile for OTP' do
+  context 'visitor can sign up and confirm a valid phone for OTP' do
     before do
       @user = sign_in_before_2fa
-      fill_in 'Mobile', with: '555-555-5555'
+      fill_in 'Phone', with: '555-555-5555'
```

Comment:
Can be done an a separate PR but I think form should say "Phone number" rather than just "Phone".